clippy_lints/src/loops/manual_flatten.rs

```diff
@@ -66,7 +76,7 @@ pub(super) fn check<'tcx>(
                     );
```

Comment:
Not an issue with your PR, but while reading yours I noticed the line above this should be `applicability` rather than `Applicability::MaybeIncorrect`. If you could fix that while you're working in this area that'd be lovely

Reply:
Okay, no problem. I'm always glad I could help here.